### ISSUE TITLE
[HIPIFY][perl] ROCm 5.7.1 on Almalinux (RHEL) has wrong path to findcode.sh 

### ISSUE THREAD

**Issue by klausbu** [ROCm-Developer-Tools/HIPIFY#1107]
I installed ROCm on Almalinux 9.2 (RHEL 9.2 "clone") and tried to hipify an application using hipconvertinplace-perl.sh which didn't work due to wrong path settings to findcode.sh which is called by hipconvertinplace-perl.sh:

/usr/bin/hipconvertinplace-perl.sh: Zeile 19: /usr/bin/../libexec/hipify/findcode.sh: Datei oder Verzeichnis nicht gefunden (this means file or directory not found)

The RHEL 9.2 installation installs findcode.sh in:

/opt/rocm-5.7.0/hip/bin
/opt/rocm-5.7.0/libexec/hipify

How should I deal with this?

**Comment by emankov** [comment]
It is known and already fixed in the ROCm HIPIFY 6.0.0 issue. 
You may fix it locally by changing:
```sh
PRIV_SCRIPT_DIR="$SCRIPT_DIR/../libexec/hipify"
```
to
```sh
PRIV_SCRIPT_DIR="$SCRIPT_DIR/../../libexec/hipify"
```
in `hipconvertinplace-perl.sh` or other scripts that are calling `findcode.sh`

I apologize for the inconvenience.

**Comment by klausbu** [comment]
Adding the following into hipconvertinplace-perl.sh works:

PRIV_SCRIPT_DIR="/opt/rocm-5.7.0/hip/bin"
